canvas_modules/common-canvas/package.json

```diff
@@ -15,7 +15,7 @@
   "license": "Apache-2.0",
   "sideEffects": false,
   "scripts": {
-    "build": "npx stylelint '**/*.scss' && grunt && rollup -c",
+    "build": "export NODE_OPTIONS=--openssl-legacy-provider && npx stylelint '**/*.scss' && grunt && rollup -c",
```

Comment:
Why is this needed here?

Reply:
Hi Matt I have added this to resolve `Error: error:0308010C:digital envelope routines::unsupported` while upgrading from node 16, alternatively we can also add this in .npmrc file.

Reply:
That's a workaround.  What is needed to fix this correctly.

Reply:
This works fine for me in common-canvas
```
Running "postcss:dist" (postcss) task
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
>> 1 processed stylesheet created.

Done.

./src/common-properties/index.js, ./src/common-properties/components/field-picker/index.js, ./src/common-properties/components/flexible-table/index.js, ./src/common-properties/controls/expression/languages/CLEM-hint.js, ./src/common-properties/controls/expression/languages/python-hint.js, ./src/context-menu/context-menu-wrapper.jsx, ./src/command-stack/command-stack.js, ./src/tooltip/tooltip.jsx, ./src/common-canvas/index.js, ./src/common-canvas/canvas-controller.js, ./src/index.js → ./dist, ./dist...
`isModuleDeclaration` has been deprecated, please migrate to `isImportOrExportDeclaration`
    at isModuleDeclaration (/Users/mhoward/github/mhoward/canvas/canvas_modules/common-canvas/node_modules/@babel/types/lib/validators/generated/index.js:2794:35)
    at PluginPass.Program (/Users/mhoward/github/mhoward/canvas/canvas_modules/common-canvas/node_modules/babel-plugin-lodash/lib/index.js:102:44)
(!) Circular dependencies
src/toolbar/toolbar-sub-menu.jsx -> src/toolbar/toolbar-sub-menu-item.jsx -> src/toolbar/toolbar-sub-menu.jsx
src/common-properties/controls/expression/expression.jsx -> src/common-properties/controls/expression/expression-builder/expression-builder.jsx -> src/common-properties/controls/expression/expression.jsx
src/common-properties/controls/control-factory.js -> src/common-properties/controls/structureeditor/index.js -> src/common-properties/controls/structureeditor/structureeditor.jsx -> src/common-properties/controls/control-factory.js
...and 15 more
created ./dist, ./dist in 11.5s
[mhoward@mbp ~/github/mhoward/canvas/canvas_modules/common-canvas (master)]$ node -v
v20.11.0
```

Reply:
I added this because it was failing during build https://github.com/elyra-ai/canvas/actions/runs/8229863100/job/22501894502#step:5:42 .